### PR TITLE
New brunch "picture-gallery" and some other tasks

### PR DESCRIPTION
**Pull request by Ekaterina1994**
Created picture gallery on page About Me and changed tags to classes in style's file. Learned methods addEventListener and forEach. Read about opacity and transparent in css. 
Read about github pages but didn't understand how to create it not in my own repository. I mean that I found info how to create site.io from my repository but can't do it from yours.

### CODE REVIEW

File: TimeWorking.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+0,5h Read about Git
+
+27.09.21
+
+12.00 – 13.30 – english
+13.00 - 14.00 – read about github pages
+14.00 – 14.15 – removed tags from css
+15.00 – 17.00 – made page About Me (read about functions and methods forEach and addEventListener + saw video and read about making pictures gallery)
+17.00 – 17.30 – read about syntax .style > .style2
+17.30 – 18.00 – read again about Git brunches and created new branch
```
Comment (tritonJS826) [inline TimeWorking.txt:17]: Choose one specific formpat and use it everywhere_

File: code/Lesson-1/aboutMe.html [hunk 2]
```diff
@@ -4,18 +4,33 @@
...
-  Illum suscipit beatae dolores corporis eos, quod dolore ullam perspiciatis facere nemo magnam quis tenetur esse tempora officiis iure dignissimos vel incidunt nostrum neque optio, sequi laudantium ratione. Illum, sed.
- </article>
+<body class="pageAboutMe">
+ <div class="container2">
+  <div class="picturesForAnimation" style="background-image: url('assets/images/boris-smokrovic-DPXytK8Z59Y-unsplash.jpg');">
+ <h3 class="nameOfBird">Bird 1</h3>
+  </div>
+  <div class="picturesForAnimation active" style="background-image: url('assets/images/boris-smokrovic-RLLR0oRz16Y-unsplash.jpg');">
+   <h3 class="nameOfBird">Bird 2</h3>
+  </div>
```
Comment (tritonJS826) [inline code/Lesson-1/aboutMe.html:17]: Semantic HTML!! where is img, picture, etc.? Be careful. Need to be fixed. It is important

File: code/Lesson-1/assets/js/scripts.js [hunk 3]
```diff
@@ -1 +1,17 @@
...
+  picturesForAnimation.classList.add('active')
+ }
+ )
+}
+
+function clearActiveClasses() {
+ slides.forEach((picturesForAnimation) => {
+  picturesForAnimation.classList.remove('active')
+ })
+}
```
Comment (tritonJS826) [inline code/Lesson-1/assets/js/scripts.js:17]: Great) we can talk about it later) now just remove alert -- it is just a mess now)

**Review by tritonJS826 — COMMENTED** [review]
Rebase and close PR after fix all remarks

File: TimeWorking.txt (hunk 1 above)
Comment (Ekaterina1994) [reply TimeWorking.txt]: Done.

File: code/Lesson-1/aboutMe.html (hunk 2 above)
Comment (Ekaterina1994) [reply code/Lesson-1/aboutMe.html:17]: Done. Found semantic tags figure (for blocks of pictures and blocks of pic plus text) and figcaption (for text).

File: code/Lesson-1/assets/js/scripts.js (hunk 3 above)
Comment (Ekaterina1994) [reply code/Lesson-1/assets/js/scripts.js:17]: Done.

**Comment by Ekaterina1994** [comment]
Fixed

**Pull request merged**